tools_webrtc/ios/build_ios_libs.py

```diff
@@ -235,15 +246,18 @@ def main():
   framework_paths = []
   all_lib_paths = []
   for (environment, archs) in list(architectures.items()):
+    ios_deployment_target = _VersionMax(
+        args.deployment_target,
+        IOS_MINIMUM_DEPLOYMENT_TARGET[environment])
     framework_path = os.path.join(args.output_dir, environment)
     framework_paths.append(framework_path)
     lib_paths = []
     for arch in archs:
       lib_path = os.path.join(framework_path, arch + '_libs')
       lib_paths.append(lib_path)
       BuildWebRTC(lib_path, environment, arch, args.build_config,
-                  gn_target_name, IOS_DEPLOYMENT_TARGET[environment],
-                  LIBVPX_BUILD_VP9, args.use_goma, args.use_remoteexec, gn_args)
+                  gn_target_name, ios_deployment_target, LIBVPX_BUILD_VP9,
+                  args.use_goma, args.use_remoteexec, gn_args)
```

Comment:
Keeping `LIBVPX_BUILD_VP9` on this line might make conflicts easier to resolve. Although this file doesn't change much, so it's not likely that we'll run into this.

Reply:
I was thinking about the upstream patch, but yeah, for our version it might make sense to not touch unnecessary lines.